analyzers/src/SonarAnalyzer.Common/AnalysisContext/SonarAnalysisContext.ScrapYard.cs

```diff
@@ -143,7 +128,7 @@ bool IsMatching(DiagnosticDescriptor descriptor)
     public static bool IsUnchanged(TryGetValueDelegate<ProjectConfigReader> tryGetValue, SyntaxTree tree, Compilation compilation, AnalyzerOptions options) =>
         UnchangedFilesCache.GetValue(compilation, _ => CreateUnchangedFilesHashSet(tryGetValue, options)).Contains(tree.FilePath);
 
-    private static ImmutableHashSet<string> CreateUnchangedFilesHashSet(TryGetValueDelegate<ProjectConfigReader> tryGetValue, AnalyzerOptions options) =>
+    private static ImmutableHashSet<string> CreateUnchangedFilesHashSet(TryGetValueDelegate<ProjectConfigReader> tryGetValue, AnalyzerOptions options) =>       // FIXME: Done
```

Comment:
Please remove these `// FIXME: Done`

Reply:
I did it as part of rebase, so it will not be in a separate commit